python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1644,6 +1629,64 @@ def test_parquet_nullable_boolean(tmpdir, engine):
     assert_eq(actual_gdf, expected_gdf)
 
 
+@pytest.mark.parametrize(
+    "pdf",
+    [
+        pd.DataFrame(index=[1, 2, 3]),
+        pytest.param(
+            pd.DataFrame(index=pd.RangeIndex(0, 10, 1)),
+            marks=pytest.mark.xfail(
+                reason="https://github.com/pandas-dev/pandas/issues/37897"
+                "https://github.com/pandas-dev/pandas/issues/37896"
```

Comment:
This was confirmed not an issue

Reply:
Should we also mimic this pandas behaviour of materializing RangeIndex in parquet writer?

Reply:
Yes, but lets do it in a follow up

Reply:
Sure, created an issue for tracking: https://github.com/rapidsai/cudf/issues/6873